cluster/manifests/instana-poc/daemonset-instana.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: instana-agent
+    version: v1
+  name: instana-agent
+  namespace: kube-system
+spec:
+  template:
```

Comment:
Should have the Rolling update strategy defined: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/skipper/daemonset.yaml#L14-L15